python/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip
 # Install the Google Cloud SDK.
 ENV HOME /
 ENV CLOUDSDK_PYTHON_SITEPACKAGES 1
+ENV CLOUD_COMPUTING 1
```

Comment:
Setting of 1 means:
- quiet output from gsutil
- multiprocessing disabled
- caching downloading files is disabled

Setting of 0 or unset means:
- verbose output
-multiprocessing enabled
- caching is enabled